### PR TITLE
Support TLS & HCP Vault PKI in the multinode env

### PR DESCRIPTION
**Pull request by Alex-Welsh**
Terraform PR: https://github.com/stackhpc/terraform-kayobe-multinode/pull/27

### CODE REVIEW

File: doc/source/configuration/vault.rst [hunk 1]
```diff
@@ -36,6 +36,8 @@ Before beginning the deployment of vault for openstack internal TLS and backend
 
   * Seed Node or a host to run the vault container on
   * Overcloud controller hosts to install second vault on
+  * Ansible Galaxy dependencies installed: ``ansible-galaxy install ansible-galaxy install -r $KAYOBE_CONFIG_PATH/anisble/requirements.yml``
```
Comment (markgoddard) [inline doc/source/configuration/vault.rst]: These should be installed using `kayobe control host bootstrap|upgrade`

File: doc/source/configuration/vault.rst [hunk 2]
```diff
@@ -36,6 +36,8 @@ Before beginning the deployment of vault for openstack internal TLS and backend
...
+  * Python dependencies installed: ``pip install -r kayobe-config/requirements.txt``
```
Comment (markgoddard) [inline doc/source/configuration/vault.rst:40]: This one's a bit awkward, I wish we had a cleaner way to pull it in.

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: doc/source/configuration/vault.rst (hunk 2 above)
Comment (Alex-Welsh) [reply doc/source/configuration/vault.rst:40]: post hook on control host bootstrap?

Comment (markgoddard) [reply doc/source/configuration/vault.rst:40]: Could do. It also includes Kayobe though, and I'd be wary of overwriting an existing installation by surprise. Would ideally like something more "official". Will tried to do it years ago but gave up for reasons I forget. I think this is ok for now.

File: doc/source/configuration/vault.rst (hunk 1 above)
Comment (Alex-Welsh) [reply doc/source/configuration/vault.rst]: I've gone with bootstrap, since I don't really know how upgrade works. Can include options for both if needed

**Review by markgoddard — APPROVED** [review]

**Pull request merged**